.eslintrc.js

```diff
@@ -0,0 +1,61 @@
+module.exports = {
+  root: true,
+  env: {
+    es6: true,
+    browser: true,
+    node: true,
+  },
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    comment: true,
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 2017,
+    sourceType: 'module',
+  },
+  settings: {
+    react: {
+      version: 'detect',
+    },
+  },
+  plugins: ['@typescript-eslint', 'react', 'react-hooks'],
+  extends: ['eslint:recommended', 'plugin:react/recommended', 'plugin:prettier/recommended'],
```

Comment:
@jtomasek the last plugins covers the integration with prettier

```suggestion
  extends: ['eslint:recommended', 'plugin:react/recommended', 'plugin:prettier/recommended'],
```